pkg/drivers/kic/oci/cli_runner.go

```diff
@@ -67,7 +67,7 @@ func (rr RunResult) Output() string {
 // PrefixCmd adds any needed prefix (such as sudo) to the command
 func PrefixCmd(cmd *exec.Cmd) *exec.Cmd {
 	if cmd.Args[0] == Podman && runtime.GOOS == "linux" { // want sudo when not running podman-remote
-		cmdWithSudo := exec.Command("sudo", cmd.Args...)
+		cmdWithSudo := exec.Command("sudo", append([]string{"-n"}, cmd.Args...)...)
```

Comment:
What is this trying to solve? Won't this break a lot of folks who require interactive use?

I suspect this should only happen if --interactive=false.

Reply:
Currently we are verifying the sudo setup when we validate the podman driver.

But since we also call it for _all_ drivers (#7963), might as well use it always ?

Reply:
As a sysadmin, I would be annoyed about a command executing things as root unnecessarily, but it's not completely clear to me how to solve #7958 elegantly without it. 

This whole business of running commands for container runtimes that have never been used is not so hot.

Reply:
Agree, it is more of a workaround...